frontend/src/types.ts

```diff
@@ -683,3 +683,8 @@ export type IntegrationAppStatus = {
   variablesValidationTimestamp?: string;
   error: string;
 };
+
+export type WarningNotification = {
```

Comment:
keep this type within the scope of hardware profiles. so move it under one of those folders in a `types.ts` folder